h2o-core/src/test/java/water/persist/PersistManagerTest.java

```diff
@@ -46,4 +46,37 @@ public void createReturnsBufferedOutputStreamForFiles() throws IOException  {
         }
     }
 
+    @Test
+    public void deltaLogFilesAreFiltered() throws IOException {
+        File dir = tmp.newFolder("to_import");
+        Files.write(new File(dir, "file-a").toPath(), Collections.singleton("file-a"));
+        Files.write(new File(dir, "file-b").toPath(), Collections.singleton("file-a"));
```

Comment:
nit: `Collections.singleton("file-b")`, but it's irrelevant for the purpose of this test

Reply:
right, I see :) will fix